wp-content/themes/pub/wporg-learn-2020/template-parts/component-breadcrumbs.php

```diff
@@ -14,20 +14,35 @@
 	),
 );
 
-// Get information about the post title.
-$cpt_object = get_post_type_object( get_post_type( get_queried_object() ) );
-
-if ( 'lesson-plan' === get_post_type() ) {
+if ( is_post_type_archive() ) {
 	array_push( $crumbs, array(
-		'label' => ucfirst( $cpt_object->labels->name ),
+		'label' => post_type_archive_title( '', false ),
+		'url' => '',
+	));
+} elseif ( is_singular( 'lesson-plan' )
+	|| is_singular( 'wporg_workshop' )
+	|| is_singular( 'course' ) ) {
+	$cpt_object = get_post_type_object( get_post_type() );
+
+	array_push($crumbs, array(
+		'label' => $cpt_object->label,
```

Comment:
remove `ucfirst` here as it's already the upper case for the first char. and it might cause problems when it comes to a different locale

Reply:
That's a good point about locales, it looks like there may be some problems w/ the multibyte ones 👍🏻